shared/types/VaccineDataTypes.ts

```diff
@@ -15,3 +15,31 @@ export type VaccineData = {
   vaccinationsPerDhb: DhbVaccineDoseData[];
   dataValidAsAtNzTimeIso: NzTimeIso;
 };
+
+type NorthIslandDhbKeys =
+  | "Northland"
```

Comment:
Why does this work? 
Is this a TS quirk?

The way I read this is `blank or "Northland" or etcetc`

Why is the blank allowed haha

Reply:
Not really blank, it is a TS/formatting quirk. Blank is not allowed

![image](https://user-images.githubusercontent.com/1582152/139792615-9fef3799-5b00-40d8-ac76-9844fc8d36a0.png)